content/docs/start/index.md

```diff
@@ -1,3 +1,8 @@
+---
+title: 'Get Started | DVC'
+description: 'Get started with DVC!'
+---
```

Comment:
Seems to work 🙂 

See https://dvc-landing-docs-descri-xlzjup.herokuapp.com/doc/start
![image](https://user-images.githubusercontent.com/1477535/94082819-6ac78000-fdc7-11ea-8ad4-89c3eaac54a3.png)

- [ ] Let's remove the changes to this file before merge.